pkg/epforwarder/epforwarder.go

```diff
@@ -138,7 +138,8 @@ func (s *defaultEventPlatformForwarder) SendEventPlatformEvent(e *message.Messag
 	case p.in <- e:
 		return nil
 	default:
-		return fmt.Errorf("event platform forwarder pipeline channel is full for eventType=%s. consider increasing batch_max_concurrent_send", eventType)
+		itemsInChannel := len(p.in)
```

Comment:
Would it make sense to use `cap` here instead? Sending to a channel only fails when the channel is full, so most of the time len will be the same as cap, except an rare case where the receiver pops a message or two out before we take the length, producing a highly confusing log message.

Reply:
I think sending to a channel can also fail if a channel has been explicitly closed elsewhere in code, but the channel being full is far more likely. I'll include both cap and length.

Reply:
Sending on a closed channel panics, even in select: https://go.dev/play/p/FbflsdxHkly